tests/cli/test_dlab.py

```diff
@@ -76,6 +78,8 @@ def setUp(self):
 
 class TestDLab(BaseCLITest):
 
+    # TODO check if sys.platform can help here for win
+    @unittest.skipIf(sys.platform == 'win32', reason="does not run on windows")
```

Comment:
Issue needs to be created and fixed in another PR

Reply:
removed